dev-lang/lua/lua-5.1.5-r106.ebuild

```
@@ -77,7 +77,7 @@ src_prepare() {
 multilib_src_configure() {
 	# We want packages to find our things...
 	sed -i \
-		-e 's:/usr/local:'${EPREFIX}'/usr:' \
+		-e 's:/usr/local:'"${EPREFIX}"'/usr:' \
```

Can we use here `"` quotes?
```suggestion
		-e "s:/usr/local:${EPREFIX}/usr:" \
```

---

done